Donate

jcolla-holla opened the pull request:
/donate page was pretty simple so I got this one out quickly.

I came up with a first round of copy for a sub-heading on this page.  Open to feedback and changes though for now I suggest we just merge this and get feedback from larger group in the future along w all the other pages.

Is the people-holding-globe image too large here?  I'm following the column guidelines of the Figma (4 columns for the globe image and max-height slightly under the px value in Figma), but still seems possibly kinda large.  Thoughts?

## Preview here:
![Screen Shot 2020-04-27 at 7 48 17 PM](https://user-images.githubusercontent.com/16062590/80431699-2095ad80-88c0-11ea-843c-8da447380f5e.png)

alexanian commented on pages/index.vue:
```diff
@@ -160,91 +125,73 @@
-      display: flex;
-      justify-content: center;
-    }
+  // this could potentially be abstracted out across other pages also.  centers CTA buttons at the bottom of each section
```
Yeah, I think we'll want to abstract out a component that is image + `<h3>` + caption + CTA button?

jcolla-holla replied:
I created a CTA component that is a <v-row> and centered consistent button that can also be passed a prop 'pdf' to show the pdf download icon (for whitepaper download specifically).  Creating an entire component for a section I don't think is a priority - there are only three such sections across the entire site currently from what I can tell.  

Maybe if we have time but want to finish all the pages first.

alexanian commented on scripts/airtable-auto.py:
```diff
@@ -0,0 +1,47 @@
+        airtable = Airtable(os.environ['AIRTABLE_BASE_KEY'], 'news', api_key=os.environ['AIRTABLE_API_KEY']) 
+        
+        # hardcoding base key and api key for now:
+        # airtable = Airtable(os.environ['app2gjpkAcqIzagxn'], 'news', api_key=os.environ['keyr0gMB3vBqrwhf0']) 
```
hmmm

jcolla-holla replied:
Whoops, I thought I had gotten rid of that line!  I reset my API key now and we can potentially use a different Base (means a different Airtable table basically) to avoid any potential risk of bad actors.

jcolla-holla commented:
- Airtable script works to pull from [this Airtable](https://airtable.com/invite/l?inviteId=invVmOLBv1X2GYuvQ&inviteToken=be7101bf6e8bd9469c83f053b372c0512c452a5f1c604ec24bf48f07b71f3346) (that I've shared w Grant to collaborate a bit on this workflow and get his perspective).  Shout out to Colby for his work on this, we basically just copied his and modified slightly.  Will need to set up Github action to do this twice a day or similar interval.

- The /news filters now work as expected (shout out to Tessa for her assistance on this!).  I added some fade in transitions for the news cards.  Preview below:

![ezgif com-video-to-gif (4)](https://user-images.githubusercontent.com/16062590/80509776-d8ba6900-8947-11ea-9d1a-52981c409343.gif)

- Added general simple fade in transition for all pages.  You can see that at the end of the gif above.  It's subtle when there is only a heading 'about rendered' but it's working.

I'm going to merge this to master now and move on to /faq.
